dev-libs/granite/granite-5.2.5-r1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+BUILD_DIR="${WORKDIR}/${P}-build"
+
+inherit meson vala xdg
+
+DESCRIPTION="Elementary OS library that extends GTK+"
+HOMEPAGE="https://github.com/elementary/granite"
+SRC_URI="https://github.com/elementary/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="doc introspection test"
```

Comment:
It works like it should! The only thing I noticed is that with a desktop profile enabled, gtk is built with introspection and this wanted to rebuild gtk+ by default. So I'd suggest adding `+introspection` here, but up to you. I'll wait for your input before merging (this contribution is good already though, so thanks!)

Reply:
Good call, fixing

Reply:
Done :)